app/services/ccms/wsdls/GetCommonOrgWsdl.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<wsdl:definitions name="GetCommonOrg" targetNamespace="http://legalservices.gov.uk/CCMS/Common/ReferenceData/1.0/GetCommonOrg"
+  xmlns:wsdl="http://schemas.xmlsoap.org/wsdl/"
+  xmlns:client="http://legalservices.gov.uk/CCMS/Common/ReferenceData/1.0/GetCommonOrg"
+  xmlns:ns1="http://legalservices.gov.uk/CCMS/Common/ReferenceData/1.0/ReferenceDataBIM"
+  xmlns:plnk="http://schemas.xmlsoap.org/ws/2003/05/partner-link/"
```

Comment:
can we name this `app/services/ccms/wsdls/GetCommonOrgDEVWsdl.xml` inline with the naming convention. I found this confusing in Apply because it was not named after the CCMS environment it was actually calling

Reply:
👍